src/CqlBuilderPanel/functionsSection/FunctionsSection.tsx

```diff
@@ -18,7 +15,10 @@ export interface FunctionProps {
   cql: string;
   isCQLUnchanged: boolean;
   functions?: FunctionLookup[];
+  resetCql?: Function;
+  cqlBuilderLookupTypes?: any;
```

Comment:
Do we need cqlBuilderLookupTypes?

Reply:
We appear to be using it in this file and passing it to consuming <FunctionBuilder and <Functions

There's probably a way to refactor it, but I avoided doing anything outside the scope of setting up the layout

Reply:
Nevermind this appears to be completely a typo one letter off of the actual variable